File: backend/src/main/java/com/tripfriend/domain/member/member/dto/AuthResponseDto.java
package com.tripfriend.domain.member.member.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class AuthResponseDto {

    private String accessToken;
    private String refreshToken;
    private boolean isDeletedAccount;

    public AuthResponseDto(String accessToken) {
        this.accessToken = accessToken;
        this.isDeletedAccount = false;
    }

    public AuthResponseDto(String accessToken, boolean isDeletedAccount) {
        this.accessToken = accessToken;
        this.isDeletedAccount = isDeletedAccount;
    }
}

File: backend/src/main/java/com/tripfriend/domain/member/member/dto/AuthResponseDto.kt
package com.tripfriend.domain.member.member.dto

data class AuthResponseDto(
    var accessToken: String? = null,
    var refreshToken: String? = null,
    var isDeletedAccount: Boolean = false
) {
    constructor(accessToken: String) : this(
        accessToken = accessToken,
        refreshToken = null,
        isDeletedAccount = false
    )

    constructor(accessToken: String, isDeletedAccount: Boolean) : this(
        accessToken = accessToken,
        refreshToken = null,
        isDeletedAccount = isDeletedAccount
    )
}